bundles/org.openhab.binding.ecovacs/README.md

```diff
@@ -0,0 +1,126 @@
+
+At this point, the following devices are fully supported and verified to be working:
+
+- Deebot OZMO 920
+- Deebot OZMO 930
+- Deebot OZMO 950
+- Deebot OZMO Slim 10/11
+
+The following devices will likely work because they are using similar protocols as the above ones:
+
+- Deebot 600/601/605
+- Deebot OZMO 610
+- Deebot 710/711/711s
+- Deebot 900/901
+- Deebot OZMO 900/905
+- Deebot OZMO T5
+- Deebot (OZMO) T8 series
+- Deebot T9 series
+- Deebot Slim 2
+- Deebot N3 MAX
+- Deebot N7
+- Deebot N8 series
+- Deebot U2 series
+- Deebot X1 Omni
+
+## Discovery
+
+At first, you need to manually create the bridge thing for the cloud API. Once that is done, the supported devices will be automatically discovered and added to the inbox.
+
+## Thing Configuration
+
+For the cloud API thing, the following parameters must be configured:
+
+| Config    | Description                                                                                                                   |
+|-----------|-------------------------------------------------------------------------------------------------------------------------------|
+| email     | The email address you used when registering the Ecovacs cloud account                                                         |
+| password  | The cloud account password                                                                                                    |
+| continent | The continent you are residing on, or 'World' if none matches. This is used to select the correct cloud server to connect to. |
+
+For the vacuum things, there is no required configuration. Optionally, you can tweak the following parameters:
+
+| Config  | Description                                                                            |
+|---------|----------------------------------------------------------------------------------------|
+| refresh | Refresh interval for polled data (see below), in minutes. By default set to 5 minutes. |
+
+## Channels
+
+The list below lists all channels supported by the binding. In case a particular channel is not supported by a given device (see remarks),
+it is automatically removed from the given thing.
+
+| Channel                          | Type                 | Description                                               | Read Only | Updated By | Remarks  |
+|----------------------------------|----------------------|-----------------------------------------------------------|-----------|------------|----------|
+| actions#command                  | String               | Command to execute                                        | No        | Event      | [1]      |
+| status#state                     | String               | Current operational state                                 | Yes       | Event      | [2]      |
+| status#current-cleaning-mode     | Number:Time          | Mode used in current cleaning run                         | Yes       | Event      | [3], [4] |
```

Comment:
Is this really a `Number:Time`?

Reply:
No, this was a copy'n'paste mistake. Thanks, fixed.